src/features/auth/hooks/useHandleForbidden.ts

```diff
@@ -0,0 +1,21 @@
+import { AxiosError } from 'axios'
+import { useRouter } from 'next/router'
+import { noAuthRoute } from '@features/auth/utils/noAuthRoute'
+import { StatusCode } from '@utils/StatusCode'
+
+export const useHandleForbidden = (error?: AxiosError) => {
+  const router = useRouter()
+
+  if (!error) return
+
+  const statusCodesMeaningForbidden = [
+    StatusCode.FORBIDDEN,
+    StatusCode.UNAUTHORIZED
+  ]
```

Comment:
This is simply wrong. You should show an Unauthorized error message when you get 401 and Forbidden error message when you get 403.